### ISSUE TITLE
Custom favicons

### ISSUE THREAD

**Issue by ccd0** [ccd0/4chan-x#1045]
from #1038:
> i also realized there's no option for using your own favicons and i think maybe it would be good to have that

**Comment by ccd0** [comment]
Requested again:
https://boards.4chan.org/g/thread/57476856

**Comment by ccd0** [comment]
Related: #437

**Comment by desaku** [comment]
I've changed the (you) favicons in the 4chanJS and xat- themes to have a red, centered exclamation mark. Do I submit the pull request as two new themes, or replace the affected icons entirely?

4chanJS
![4chanjs you](https://cloud.githubusercontent.com/assets/23438357/20248908/3eedc7a4-a9bc-11e6-9233-8b25e7212be9.png)
xat-
![xat you](https://cloud.githubusercontent.com/assets/23438357/20248910/440427b0-a9bc-11e6-8588-d7eccbb78ecc.png)

**Comment by ccd0** [comment]
Is red what they look like in the native extension? If so, I would replace them for 4chanJS. For the xat- theme, which didn't have an exclamation mark to begin with, it would probably make more sense to create a new theme.

**Comment by ccd0** [comment]
Also, I think the native has a white exclamation mark for new posts and a red exclamation mark for when you get replied to. Although that will take code changes to implement rather than just changing the icons.

**Comment by ccd0** [comment]
Oops, wasn't looking at the right icons when I wrote some of that. I still think that updating 4chanJS to agree with native makes sense, but an altered xat- should be a new theme.

**Comment by ccd0** [comment]
> wasn't looking at the right icons when I wrote some of that.
Speaking of which: #1191

**Comment by desaku** [comment]
Yes, they look red in the native extension. The white exclamation mark is still there for threads with new replies but no (you)s. I altered the xat theme because its (you) favicon appeared to have a resolution problem, so I'll submit that alteration as a new theme (xat-2).

xat- (unread, Old unread (you), New unread (you))
![screenshot_20161113_181104](https://cloud.githubusercontent.com/assets/23438357/20249694/de89e472-a9cc-11e6-943a-87f292b4c82c.png)

**Comment by ccd0** [comment]
> I altered the xat theme because its (you) favicon appeared to have a resolution problem, so I'll submit that alteration as a new theme (xat-2).

If you're just doing it to fix a bug instead of because you want the new look, let's leave the look as is and fix the bug.

**Comment by desaku** [comment]
So have the xat- icons been fixed, or should I submit a new theme? Currently cloning the repo locally now.

**Comment by ccd0** [comment]
They should be. The only issue was that they were 14x13 instead of 16x16, causing them to be resized by the browser. I just added a transparent border like on the others.
